src/components/Table/GridTable.test.tsx

```diff
@@ -1023,6 +1023,26 @@ describe("GridTable", () => {
     // Then we still get back the header row
     expect(rows).toEqual([simpleHeader]);
   });
+
+  it("can apply emptyCell to missing content", async () => {
+    // Given the table with a specified emptyCell
+    const r = await render(
+      <GridTable<Row>
+        columns={[nameColumn, valueColumn]}
+        style={{ emptyCell: <>empty</> }}
+        rows={[
+          simpleHeader,
+          // And some content is undefined and empty strings
+          { kind: "data", id: "2", name: "", value: 2 },
+          { kind: "data", id: "1", name: "a", value: undefined },
+          { kind: "data", id: "3", name: "c", value: 1 },
```

Comment:
Kind of a nit, but these rows are probably un-ordered from a client-side sorting test; maybe order them to just be slightly easier to follow...

Reply:
Also what if `value` was `0`? Would we replace that b/c it's falsely? Should we?

It seems like `null` / `undefined` / even `""` should definitely be replaced with `emptyCell` ... not sure about `0` though...seems like a wrinkle...

Reply:
Agreed. I think 0 should be shown if that is the value.